src/main/scala/beam/agentsim/agents/household/HouseholdFleetManager.scala

```diff
@@ -45,7 +45,7 @@ class HouseholdFleetManager(parkingManager: ActorRef, vehicles: Map[Id[BeamVehic
           veh.spaceTime = SpaceTime(homeCoord.getX, homeCoord.getY, 0)
           veh.mustBeDrivenHome = true
           for {
-            ParkingInquiryResponse(stall, _) <- parkingManager ? ParkingInquiry(homeCoord, "home", 0.0, None, 0)
+            ParkingInquiryResponse(stall, _) <- parkingManager ? ParkingInquiry.apply(homeCoord, "home")
```

Comment:
Same `apply` comment:

```suggestion
            ParkingInquiryResponse(stall, _) <- parkingManager ? ParkingInquiry(homeCoord, "home")
```

Reply:
not my code, but removed it